### PR TITLE
Scorecard player builder creates players

### PR DESCRIPTION
**Pull request by ChaeOkay**
The ScorecardPlayerBuilder creates players.
This PR replaces the Player#new and Player#create actions with the new service object.

Next PR will attach players to the scorecard.

### CODE REVIEW

**Comment by ChaeOkay** [comment]
@DavidOKeefe :eyeglasses:

File: app/controllers/scorecard_player_builders_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class ScorecardPlayerBuildersController < ApplicationController
+  respond_to :html
```
Comment (DavidOKeefe) [inline app/controllers/scorecard_player_builders_controller.rb]: What's this for?

**Comment by DavidOKeefe** [comment]
I had one question, but looks good!

File: app/controllers/scorecard_player_builders_controller.rb (hunk 1 above)
Comment (ChaeOkay) [inline app/controllers/scorecard_player_builders_controller.rb]: Ah, this tells rails that every action in this controller is going to respond only to an html request (as opposed to other mime type requests like json or xml). 

By doing this, it allows you to use some different syntax within the controller. I thought I wanted it, but it's totally unnecessary.

Line 2 has been removed!

**Pull request merged**